pipeline/customer/customer_test.py

```diff
@@ -342,7 +343,40 @@ def test_nonempty(self):
             customer.Field.Errors: [],
         })
         expected = ['{"errors": [], "first_name": "Winny", "id": 3, "last_name": "Tuppeny", "sum": "1075.90"}']
-        actual = [data] | beam.ParDo(JsonFormatter())
+        actual = [data] | beam.ParDo(JsonFormatter(StaticValueProvider(unicode, 'sum')))
+        self.assertEqual(expected, actual)
+
+    def test_nonempty_avg(self):
```

Comment:
To be complete, we would need tests for the other aggregations, too. For the sake of this training, sum and avg are probably sufficient.

Reply:
Correct. There is no benefit for the training.